src/utils.c

```diff
@@ -367,6 +367,31 @@ int map_intd(char *s, char **v, int dv) {
     return n;
 }
 
+int maps_intd(char *s, char **v, int dv) {
```

Comment:
Please find a more suggestive name for the function

Reply:
Any suggestion? I selected this as `map_intd()` is transformed to `map_special_intd()` and contracted to `maps_intd()`. What you want? `map_with_strcasecmp_intd()` or something similar?

Reply:
search_string_in_array?

Reply:
Hi @catalinii ,

Before change it, I want to be sure: You agree with the name `check_strs()` ?